config/jail.conf

```diff
@@ -344,7 +344,7 @@ logpath = %(lighttpd_error_log)s
 [roundcube-auth]
 
 port     = http,https
-logpath  = /var/log/roundcube/userlogins
+logpath  = /var/log/roundcubemail/errors
```

Comment:
I did some digging into roundcube's source, it seems logging failed login was added in 2013: https://github.com/roundcube/roundcubemail/commit/060467df9d95be3768dab51ff5dd4e6214ec86a0 - which is tagged as 1.0-rc.

Assuming we don't plan on having fail2ban support pre-1.0 roundcube, the plugin you referenced is not needed.

Roundcube logs are stored within its content directory by roundcube default (/logs). The .deb and .rpm configure `log_dir`. This is from the .deb (not direct configuration, but a clue nonetheless):
```
./postinst:	chown -R www-data:adm /var/log/roundcube
./postinst:	chmod -R 750 /var/log/roundcube
./postrm:	rm -rf /var/log/roundcube
```
I don't have a test Debian box to use, but several points show /var/log/roundcube is used.

On CentOS the defaults.inc.php includes `$config['log_dir'] = '/var/log/roundcubemail/';`.

So, as I mentioned before, it makes sense to use these two directories as variables in paths-*.

However, the optional configuration (`$config['log_logins'] = true;`) would still need to be added manually to enable the 'userlogins' log (instructions for which I also added to the filter config). Whereas the 'errors' log is always present (hence default fail2ban matching default roundcube). That is the crux of my justification for changing the default `logpath` (conceding the directory be a configuration per platform).

Assuming we do change the default `logpath`, how can it be made less vague? Change log says it changed, fail2ban will fail to start after upgrade if the `logpath` is not valid (that is pretty obvious imho). Point being fail2ban won't silently stop working as a result; with proper change control (did it start after you upgraded?) or monitoring (is fail2ban running?) it should be more than evident to an administrator. But if more clarity can be added somewhere specifically, I would be more than happy to.